### PR TITLE
Sam/Repo Sync Test

### PR DESCRIPTION
**Pull request by samholmes**
Rename "stress test" to "repo sync test" and reorg utility code for reuse in other bins (e.g. the new load test)

### CODE REVIEW

File: src/bin/utils/console.ts [hunk 1]
```diff
@@ -12,7 +12,7 @@ const rl = createInterface({
 rl.on('line', line => {
   const log = JSON.parse(line)
 
-  if (log.msg === 'status') {
+  if (log.msg === 'console') {
```
Comment (swansontec) [inline src/bin/utils/console.ts:15]: Typically, rename commits should not make code changes at the same time. No need to fix, just a note for the future.

File: src/bin/repo-sync-test/worker-routine.ts [hunk 2]
```diff
@@ -28,10 +28,8 @@ let isRepoSynced: boolean = true
 export async function workerRoutine(config: WorkerConfig): Promise<void> {
   // Create sync clients
   const syncClients = Object.entries(config.clusters).reduce<SyncClient[]>(
-    (syncClients, [clusterName, urls]) => {
-      const clients = urls.map(
-        serverUrl => new SyncClient(serverUrl, clusterName)
-      )
+    (syncClients, [_, urls]) => {
```
Comment (swansontec) [inline src/bin/repo-sync-test/worker-routine.ts]: If `_` isn't used, wouldn't it be `Object.values`?

**Review by swansontec — APPROVED** [review]
Approved, with questions.

File: src/bin/utils/console.ts (hunk 1 above)
Comment (samholmes) [reply src/bin/utils/console.ts:15]: Bugger.. I thought I was careful about refactoring this out. Doubly noted.

File: src/bin/repo-sync-test/worker-routine.ts (hunk 2 above)
Comment (samholmes) [reply src/bin/repo-sync-test/worker-routine.ts]: Nice catch. I was too quick in my approach here. I'll fix it.

**Pull request merged**